spec/index.html

```diff
@@ -1462,58 +1462,37 @@ <h3>The <code>rdf:JSON</code> Datatype</h3>
       <dt id="JSON-lexical-space">The <a>lexical space</a></dt>
       <dd>is the set of all <a data-lt="string">RDF strings</a> that conform to the
         <a data-cite="RFC8259#section-2">JSON Grammar</a> as described in
-        <a data-cite="RFC8259#section-2">Section&nbsp;2 JSON Grammar</a> of
-        [[RFC8259]].</dd>
+        <a data-cite="RFC8259#section-2">Section&nbsp;2 JSON Grammar</a> of [[RFC8259]],
+        which also conform to the requirements of [[[RFC7493]]] [[RFC7493]].</dd>
```

Comment:
So the value space includes 1.0 and 1.00 as different elements.

Reply:
If the value space is the result of JCS, then they there is a single normative representation, which would map to `1`. The serialization is defined in pseudo-code in [ECMASript 7.1.12.1](https://262.ecma-international.org/10.0/#sec-tostring-applied-to-the-number-type).

However, in the lexical space, they would be considered different terms.

If the value space is based on a native representation, then `1.0` and `1.00` (and `1`) all result in the same IEEE floating point number.

Reply:
A JSON number is a string (or maybe a parse tree), where 1.0 and 1.00 are different.   If you want the value space here to be IEEE floating point double you have to say so.